### ISSUE TITLE
JSON RPC incorrectly reports recording PVR status

### ISSUE THREAD

**Issue by monochromec** [xbmc/xbmc#24999]
Omega's JSON API incorrectly reports PVR recording status
### Describe the bug

The following Python3 snippet reports an incorrect recording status:
```# Check if a backend is recording                                                                                                                                                                         
    def checkRecording(self):
	try:
           # Assumes an existing API instance in self._kodi
            _rec = self._kodi.PVR.GetProperties({'properties':['recording']})['result']['recording']
            logging.debug('PVR recording: %d', _rec)
        except:
            return False
        return _rec
```
## Expected Behavior
_rec should be False if not recording

## Actual Behavior
_rec is True regardless of PVR recording state

### To Reproduce
1. Start Omega
2. Run above snippet on the instance API
Used Operating system:
 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [X] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Noble Numbat (beta)
 - Kodi version: Omega

## Debug Log 
N/A

##Screenshots
N/A

More than happy to provide additional information, just let me know.

**Comment by ksooo** [comment]
Does this work properly in Nexus?

**Comment by ksooo** [comment]
I'm not an expert how to use the Python API, but is `_rec` not the error code from the call, only indicating whether the call went okay, but this is not the actual data you want!

I did a quick test and APIs eems to work like it should:

When not recording I get:
```
curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":"1","method":"PVR.GetProperties","params":{"properties":["recording"]}}' http://kodi:kodi@localhost:8080/jsonrpc
{"id":"1","jsonrpc":"2.0","result":{"recording":false}}
```
Please note the "recording:false" in the response.

When recording, response looks different:
```
curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":"1","method":"PVR.GetProperties","params":{"properties":["recording"]}}' http://kodi:kodi@localhost:8080/jsonrpc
{"id":"1","jsonrpc":"2.0","result":{"recording":true}}
```

For me it seems that it works as it should. Please double check your python code is correct to not check for the error code of the call but the actual property value.

**Comment by monochromec** [comment]
> Does this work properly in Nexus?

Yes, only after the upgrade to Noble and Omega (which I compiled myself due to the Python 3.12 issue with the version of from Noble repo) the issue above surfaced. 

Even stranger: I cannot reproduce this any longer... 🤔
Not sure if that's a fluke as I traced back my steps w/o success. So closing this issue; should this resurface, I'll open another issue.

@ksooo : Thanks for quick feedback, much appreciated. For your benefit: 
```self._kodi.PVR.GetProperties({'properties':['recording']})['result']['recording']```
extracts the result key including the value from the dictionary returned (mapped from JSON object). If this value is False, so is ```_rec```, so you can simply check this in a corresponding if-statement to get the status of the recording...
